sql/updates/hotfixes/master/2017_99_99_18010_hotfixes.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS `conversation_line`;
+CREATE TABLE `conversation_line` (
+  `Id` int (10) UNSIGNED NOT NULL,
+  `Unk1` int (10) UNSIGNED NOT NULL,
+  `Unk2` int (10) UNSIGNED NOT NULL,
+  `Unk3` int (10) UNSIGNED NOT NULL,
+  `NextLineID` smallint (5) UNSIGNED NOT NULL,
+  `Unk4` smallint (5) UNSIGNED NOT NULL,
+  `UnkBoolean` tinyint (3) UNSIGNED NOT NULL,
+  `Unk5` tinyint (3) UNSIGNED NOT NULL,
+  `Unk6` tinyint (3) UNSIGNED NOT NULL,
```

Comment:
one of these unknowns is a string field, but the dbcviewer does not show any useful things